stock_demand_estimate/models/stock_demand_estimate.py

```diff
@@ -62,7 +62,10 @@ class StockDemandEstimate(models.Model):
     @api.depends('product_qty', 'date_range_id.days')
     def _compute_daily_qty(self):
         for rec in self:
-            rec.daily_qty = rec.product_qty / rec.date_range_id.days
+            if rec.date_range_id.days:
```

Comment:
I'm not sure about this fix, if you have days = 0 is because of a missconfiguration on the date ranges.

Reply:
Ok, I put the <code>create="0"</code> in the tree view.